benchmark/torch/mappo/env_wrappers.py

```diff
@@ -0,0 +1,103 @@
+"""
+Modified from OpenAI Baselines code to work with multi-agent envs
+"""
+import numpy as np
+from mpe.environment import MultiAgentEnv
+from mpe.scenarios import load
+
+
+def MPEEnv(scenario_name, rank, seed):
+    # load scenario from script
+    scenario = load(scenario_name + ".py").Scenario()
+    # create world
+    world = scenario.make_world()
+    # create multiagent environment
+    env = MultiAgentEnv(world, scenario.reset_world, scenario.reward,
+                        scenario.observation, scenario.info)
+    env.seed(seed + rank * 1000)
+    return env
+
+
+class ParallelEnv(object):
```

Comment:
please replace ParallelEnv with single-environment Wrapper and VecWrapper (https://github.com/PaddlePaddle/PARL/blob/develop/parl/env/vector_env.py)